src/test/java/seedu/address/logic/commands/FindByPhoneCommandTest.java

```diff
@@ -82,8 +82,6 @@ public void execute_multipleKeywords_multiplePersonsFound() {
         PhoneContainsKeywordsPredicate predicate = preparePredicate("9482 563");
         FindByPhoneCommand command = new FindByPhoneCommand(predicate);
         expectedModel.updateFilteredPersonList(predicate);
-        System.out.println(command);
-        System.out.println(expectedModel.getFilteredPersonList().toString());
```

Comment:
Good to delete redundant printing statement.

Reply:
Thanks bb